default.nix

```diff
@@ -1,4 +1,4 @@
-{ pkgs ? import <nixpkgs> {} }:
-{
-  agenix = pkgs.callPackage ./pkgs/agenix.nix  {};
-}
+{ pkgs ? import <nixpkgs> { } }:
+
+pkgs.callPackage ./pkgs/agenix.nix { }
+
```

Comment:
I think it is a breaking change to change the return type of default.nix, so I definitely want to avoid this.